dev/manual_tests/card_collection.dart

```diff
@@ -324,7 +324,7 @@ class CardCollectionState extends State<CardCollection> {
         child: new SingleChildScrollView(
           child: new Container(
             height: cardModel.height,
-            decoration: new BoxDecoration(backgroundColor: theme.primaryColor),
+            color: theme.primaryColor,
```

Comment:
I meant keep the Container.backgroundColor name. Since this will always refer to the background color of the Container.